homeassistant/components/media_player/sonos.py

```diff
@@ -71,6 +71,9 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     import soco
     global DEVICES
 
+    if hasattr(hass.config, 'host'):
```

Comment:
I think we should also use this in `get_local_ip`on `homeassistant.util` and the default should be to autodetect the ip (like we do today).